Back End/MemorizeWords/MemorizeWords/Infrastructure/Utilities/ValidationTypeUtility.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MemorizeWords.Infrastructure.Transversal.Exception.Exceptions;
+
+namespace MemorizeWords.Infrastructure.Utilities;
+
+public class ValidationTypeUtility
+{
+    public static void ThrowIfNullOrDefault<T>(T value, string settingsName)
```

Comment:
string settingsName parameter shouldnt be for spesific. This is a utility function so we don't know what is the purpose of use of this method by programmers in the future. We cannot throw static message here

Change string settingsName to messageName and only throw the given messageName parameter as exception